### PR TITLE
Team Page Addition

### PR DESCRIPTION
**Pull request by dpaulos6**
Added team page to the project (Updated `staffs.json` to the most recent).
Adjusted the width of the "Unleash the fun" button on the home page.
Updated `next.config` to accept the required URL's for the staff avatars.
Updated `favicon` to Would You Bot's icon.

Edit:
Updated the footer to align the links at the top rather than vertically centered.

### CODE REVIEW

File: data/staffs.json [hunk 1]
```diff
@@ -1,225 +1,194 @@
...
+    "categories": ["Contributors"],
+    "description": "Frontend Expert",
+    "imageUrl": "https://cdn.wouldyoubot.gg/staff/special/Paulos.webp",
+    "websiteUrl": "https://dpaulos6.xyz?utm_source=wouldyoubot.gg"
+  },
+  {
+    "name": "Paul",
+    "id": 24,
+    "categories": ["Developers"],
+    "description": "Web Developer",
```
Comment (Jester0027) [inline data/staffs.json:190]: Come on man
```suggestion
    "description": "Full-stack Developer",
```

File: app/team/page.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                {role}
+              </h2>
+              <ul
+                key={i}
+                role="list"
+                className="text-pretty mx-auto mt-10 flex max-w-2xl flex-wrap justify-center gap-6 px-6 lg:mx-0 lg:max-w-none lg:gap-8 lg:px-8"
+              >
+                {staffs
+                  .filter((staff) => staff.categories.includes(role))
+                  .map((staff) => (
```
Comment (Jester0027) [inline app/team/page.tsx:53]: An array reduce would be clean imo, but that's up to you if you actually want to do that 😄

File: data/staffs.json
Comment (Jester0027) [inline data/staffs.json:1]: Why were some people removed here ?

Comment (Jester0027) [reply data/staffs.json:1]: My bad, I just can't read

**Review by Jester0027 — CHANGES_REQUESTED** [review]

File: app/team/page.tsx (hunk 2 above)
Comment (dpaulos6) [reply app/team/page.tsx:53]: this is what they made on the website, I just added it to the page, I would structure the files in a different way but I won't change how Dominik made it.

**Review by Jester0027 — APPROVED** [review]

**Pull request merged**